MVP/defaults-build.yaml

```diff
@@ -12,18 +12,20 @@
             script-only-if-succeeded: False
             script-only-if-failed: True
             builders:
-              - shell: |
-                  #!/bin/bash
-                  echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
-                  bkr job-cancel ${{BKR_JOBID}}
+                - shell: |
+                    echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
+                    bkr job-cancel ${{BKR_JOBID}}
         - archive:
             artifacts: '*.txt, *.xml, *.properties'
             allow-empty: 'true'
         - junit:
             results: 'J:*.xml'
         - email-ext:
             recipients: ${{OWNERSHIP,var="JOB_COOWNERS_EMAILS"}}
-            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}]
+            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}] (${{TEST_COUNTS,var="fail"}}/${{TEST_COUNTS}})
+            body: |
+                $DEFAULT_CONTENT
+                Beaker job at https://beaker.engineering.redhat.com/matrix/?job_ids=${{ENV,var="BKR_JOBID_NUM"}}
```

Comment:
Good advice.
Links are not clickable on some clients by default (like mine), so wasn't aware of that.

BTW another func update is also necessary before we can call it multi-job supported.
Will open another PR for that - and fix matrix URL for multi-ID there as well.